links/gql_http_link/lib/src/link.dart

```diff
@@ -32,13 +32,21 @@ class HttpLinkResponseContext extends ContextEntry {
   /// HTTP status code of the response
   final int statusCode;
 
+  /// HTTP response headers
+  final Map<String, String> headers;
+
   const HttpLinkResponseContext({
     @required this.statusCode,
-  }) : assert(statusCode != null);
+    @required this.headers,
+  }) : assert(
+          statusCode != null,
```

Comment:
Thanks for this. Maybe we can add some explicit error messages here?

Reply:
I have followed the same assert style on the project, split it in two lines and keep it without error message.

Reply:
Right now this checks `statusCode != null` with the message `headers != null` should be:
```dart
assert(statusCode != null),
assert(headers != null);
```
The messages here would be too generic to be worth including IMO

Reply:
I fixed this mistake.